test/client/App.test.js

```diff
@@ -4,9 +4,10 @@ import renderer from 'react-test-renderer';
 import App from '../../client/components/App';
 
 describe('App component', () => {
-  it('renders', () => {
+  it('renders all three sections', () => {
```

Comment:
what does 'all three sections' mean? all three sections of the reservations module?
@poema-cavalcanti